test/to-port-to-wpts/misc2.js

```diff
@@ -974,10 +975,10 @@ describe("jsdom/miscellaneous", () => {
     });
 
     specify("ensure_resources_can_be_skipped_via_options_features", { async: true }, t => {
-      const html = `<html><head><script src="./files/hello.js"></script>
-                 <script src="./files/nyan.js"></script></head>
+      const html = `<html><head></head>
                  <body><span id="test">hello from html</span><span id="cat">
-                 hello from cat</body></html>`;
+                 hello from cat</body><script src="./files/hello.js"></script>
```

Comment:
scripts should be before `</body>`, ideally.

Reply:
Fixed